### PR TITLE
Cleanup met files

### LINKED ISSUE #6863: Met files in Examples folder have got messy and there are errors
There are met files in there that have errors and others that are not helpful (e.g. with coordinates for names). Suggestions:
- all files retained should be named as CC_LocationName with CC being the usual country code 
- all files retained should be tested with an end to end run of a simple simulation (SoilWater and no crops?) to test for errors
- met files to demonstrate different ways of doing met files should be in the Tutorial folder (and should be short)
- junk files should be deleted to reduce cruft

### PR DESCRIPTION
**Pull request by ric394**
resolves #6863 

1. Renames met files with country code and place name.
2. Adds converter.
3. Resource files updated to 172

### CODE REVIEW

File: .vscode/launch.json
Comment (par456) [inline .vscode/launch.json:1]: can you pull this one out of this PR

File: ApsimNG/ApsimNG.csproj
Comment (par456) [inline ApsimNG/ApsimNG.csproj:1]: can you pull this one out of this PR

**Review by par456 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json
Comment (ric394) [reply .vscode/launch.json:1]: I've updated the PR so they are the same, this shouldn't cause and issue. Tell me if this not adequate.

File: ApsimNG/ApsimNG.csproj
Comment (ric394) [reply ApsimNG/ApsimNG.csproj:1]: I've updated the PR so they are the same, this shouldn't cause and issue. Tell me if this not adequate.

File: Models/Core/ApsimFile/Converter.cs
Comment (par456) [inline Models/Core/ApsimFile/Converter.cs:1]: Git has buggered this file up. Grab your function out, reset the file, and your function back in again.

Also, the filename part just needs to be more specific, add "/Examples/WeatherFiles/" to the start of each thing you search for and replace.

File: ApsimNG/Resources/TreeViewImages/ObservedReport.svg
Comment (par456) [inline ApsimNG/Resources/TreeViewImages/ObservedReport.svg:1]: Why is this in here?

**Review by par456 — APPROVED** [review]

**Pull request merged**